_docs/zh/administration/manual-compact.md

```diff
@@ -138,9 +140,57 @@ for example:
            --target_level 2 --bottommost_level_compaction force
 ```
 
-该工具不仅会调用shell命令设置Table环境变量，对于once类型还会等待所有的replica上的操作都执行完成，使用起来十分方便。
+该工具不仅会调用 shell 命令设置 Table 环境变量，对于 once 类型还会等待所有的 replica 上的操作都执行完成，使用起来十分方便。
 
-譬如，在[bulk load](usage-scenario#支持场景)完成后执行once manual compact如下：
+譬如，在 [bulk load](usage-scenario# 支持场景) 完成后执行 once manual compact 如下：
 ```
 $ ./scripts/pegasus_manual_compact.sh -c 127.0.0.1:34601,127.0.0.1:34602 -a temp
 ```
+
+
+
+## 通过 admin-cli 设置
+
+在 **2.4.0** 之后的 pegasus 版本还支持用 admin-cli 来设置 manual compaction 的开始并且可以方便的查看进行的进度。
+
+### 使用命令
+
+```bash
+#开始单次 manual compaction
+Pegasus-AdminCli-1.2.0 » manual-compaction start -h
+start manual compaction for a specific table
+
+Usage:
+  start [flags]
+
+Flags:
+  -b, --bottommostLevelCompaction           bottommost level files will be compacted or not, default value is false
+  -h, --help                                display help
+  -c, --maxConcurrentRunningCount int       max concurrent running count, default value is 0, no limited (default: 0)
+  -a, --tableName                 string    table name
+  -l, --targetLevel               int       compacted files move level, default value is -1 (default: -1)
+  
+  
+#查看 manual compaction 进度
+Pegasus-AdminCli-1.2.0 » manual-compaction query -h
+query manual compaction progress for a specific table
+
+Usage:
+  query [flags]
+
+Flags:
+  -h, --help                display help
+  -a, --tableName string    table name
+
+```
+
+
+
+## 补充说明
+
+manual compaction 常与 bulk load 功能配合使用，作为批量导入大量数据后统一优化读取的手段。在需要进行 bulk load 操作的表中，我们常将 **Usage Scenario** 参数设置为 bulk_load 模式，以便减小增加大量数据带来的性能损耗。
```

Comment:
Better to write the docs objectively, avoid to say "We often do ...", because this is an open source project, who is we?